src/go/k8s/internal/controller/redpanda/redpanda_controller.go

```diff
@@ -745,6 +802,11 @@ func (r *RedpandaReconciler) createHelmReleaseFromTemplate(ctx context.Context,
 			Retries:  1,
 			Strategy: &rollBack,
```

Comment:
Do you happen to know if DisableWait will effectively disable Remediation or would it make remediation more aggressive?

Reply:
I think, but don't really know without figuring out some sort of way of testing it, that the only way remediation will happen is if something in the fire and forget process is just straight up broken -- i.e. the helm chart template fails to get rendered properly or if, even without waiting for the jobs/rolling updates to complete, the helm chart still somehow manages to time out the upgrade.

So, in other words, I think that this will effectively "disable" remediation unless something super unexpected and bad happens.

Reply:
Do you think it'd be worth disabling the remediation, formally, so we don't have to worry about any gotchas in flux popping up unexpectedly?

Reply:
Maybe? I suppose it'd take us one step closer to just "apply all the resources and see what happens" world. I'd imagine at least in the case of a failed helm template render we'd get something like:

1. The `HelmRelease` is marked as failed
2. The `Redpanda` is then marked as `Ready = False`
3. If the helm render failed we'd get the `StatefulSet` of the cluster never having gotten touched, so all of the nodes would remain as is.

In the non-flux world I'd imagine we'd do something similar on an upgrade:

1. We got an error running the helm rendering pipeline, mark the `Redpanda` instance as `Ready = False` or whatever else makes sense
2. Don't touch the `StatefulSet`/`Deployment` or other resources because this was a rendering error that needs someone to intervene in the CRD configuration and until then we keep plugging away.

Reply:
Done